### PR TITLE
Add tabs to code examples with object format.

### LINKED ISSUE #1674: Code examples with tabs
Let's have some api to create code examples with tabs on top of it.

Here is a real example:
image

Let's change it to
```ts
block.code({
   'npm': 'npm install @vuestic/agGrid-theme',
   'yarn': 'yarn add @vuestic/agGrid-theme'
})
```
I'm talking about this kind of tabs:
image

### PR DESCRIPTION
**Pull request by xiongmao86**
## Description
Add tabs to code examples.
fixes #1674

## Markup:
Pass in code object with keys as tab, values as code content.
<details>
```vue
block.code({
   'npm': 'npm install @vuestic/agGrid-theme',
   'yarn': 'yarn add @vuestic/agGrid-theme'
})
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/DocsCode.vue
Comment (asvae) [inline packages/docs/src/components/DocsCode.vue:35]: You can replace `Array<string>` with `string[]`. Does the same, looks better :).

**Comment by asvae** [comment]
Thanks!

Can we apply this docs feature for some of the docs as well?

The immediate problem I see is that docs configs declaration doesn't have tabs defined in types. So right now we would have to immediately follow your effort up, so that it doesn't become dead code. :)

Makes sense?

File: packages/docs/src/components/DocsCode.vue
Comment (xiongmao86) [reply packages/docs/src/components/DocsCode.vue:35]: Absolutely.

File: packages/docs/src/components/DocsCode.vue
Comment (RVitaly1978) [inline packages/docs/src/components/DocsCode.vue:44]:
```suggestion
    const tabs = computed(() => isString.value ? [] : Object.keys(props.code))
```

File: packages/docs/src/components/DocsCode.vue
Comment (RVitaly1978) [inline packages/docs/src/components/DocsCode.vue:56]:
```suggestion
    const contents = computed(() => isString.value
      ? [applyTranslations(props.code.trim())]
      : tabs.value.map(tab => applyTranslations(props.code[tab].trim()))
    )
```

File: packages/docs/src/components/DocsCode.vue
Comment (RVitaly1978) [inline packages/docs/src/components/DocsCode.vue:58]:
```suggestion
    const index = ref(0)
```

File: packages/docs/src/components/DocsCode.vue
Comment (RVitaly1978) [inline packages/docs/src/components/DocsCode.vue:13]:
```suggestion
    :code="contents[index]"
```

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]

**Comment by xiongmao86** [comment]
> Thanks!
> 
> Can we apply this docs feature for some of the docs as well?
> 
> The immediate problem I see is that docs configs declaration doesn't have tabs defined in types. So right now we would have to immediately follow your effort up, so that it doesn't become dead code. :)
> 
> Makes sense?

Ah, I see it now. Thanks.

File: packages/docs/src/components/DocsCode.vue
Comment (xiongmao86) [reply packages/docs/src/components/DocsCode.vue:13]: `model:value` of `va-tabs` start with 1, but contents start with 0, I have deliberately checked this out in demo book that start with `yarn serve`, so I think `index` should probably stay the same.

Comment (RVitaly1978) [reply packages/docs/src/components/DocsCode.vue:13]: I guess there is a bug in our `VaTabs` component with tab indexes. So, I opened a PR #1702 . Let's wait for his approval.

Comment (xiongmao86) [reply packages/docs/src/components/DocsCode.vue:13]: Sure, thank you @RVitaly1978.

Comment (RVitaly1978) [reply packages/docs/src/components/DocsCode.vue:13]: PR https://github.com/epicmaxco/vuestic-ui/pull/1702 has been approved and merged into the develop branch. So now `model:value` of `va-tabs` is equal to its index. You can apply this suggestion)

File: packages/docs/src/components/DocsCode.vue
Comment (RVitaly1978) [reply packages/docs/src/components/DocsCode.vue:41]: see as above

**Review by RVitaly1978 — CHANGES_REQUESTED** [review]

**Comment by xiongmao86** [comment]
This is ready for review.

**Review by m0ksem — APPROVED** [review]
Thanks! 🎉

Checked with  https://github.com/epicmaxco/vuestic-ui/pull/1702.

File: packages/docs/src/page-configs/extensions/ag-grid/page-config.ts
Comment (xiongmao86) [inline packages/docs/src/page-configs/extensions/ag-grid/page-config.ts:31]: I think I have update [icon-fonts](https://vuestic.dev/en/getting-started/configuration-guide#icon-fonts) here.

File: packages/docs/src/page-configs/extensions/ag-grid/page-config.ts
Comment (xiongmao86) [inline packages/docs/src/page-configs/extensions/ag-grid/page-config.ts:4]: I think I have update [ag-grid#ag-grid-installation](https://vuestic.dev/en/extensions/ag-grid#ag-grid-installation) here.

File: packages/docs/src/page-configs/getting-started/configuration-guide/code-examples/icons.ts
Comment (xiongmao86) [inline packages/docs/src/page-configs/getting-started/configuration-guide/code-examples/icons.ts:1]: And update [configuration-guide#icons-fonts](https://vuestic.dev/en/getting-started/configuration-guide#icon-fonts)

**Comment by xiongmao86** [comment]
Thank you for helping me with this pr, @asvae @RVitaly1978 @m0ksem.

**Pull request merged**
